### PR TITLE
135 update parallel navigation component

### PR DESCRIPTION
**Pull request by kjl16**

### CODE REVIEW

File: packages/patternlab/source/patterns/molecules/parallel-navigation-item/parallel-navigation-item.twig [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{% include '@molecules/parallel-navigation-item/parallel-navigation-item.twig' with {
+  active: false,
+  link_color: 'beaver-blue',
```
Comment (lal65) [inline packages/patternlab/source/patterns/molecules/parallel-navigation-item/parallel-navigation-item.twig:3]: Kill `link_color`.

File: packages/patternlab/source/patterns/molecules/parallel-navigation-item/parallel-navigation-item~active.twig [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{% include '@molecules/parallel-navigation-item/parallel-navigation-item.twig' with {
+  active: true,
+  link_color: 'beaver-blue',
```
Comment (lal65) [inline packages/patternlab/source/patterns/molecules/parallel-navigation-item/parallel-navigation-item~active.twig:3]: Kill `link_color`.

File: packages/patternlab/source/patterns/molecules/parallel-navigation-item/parallel-navigation-item~related-active.twig [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{% include '@molecules/parallel-navigation-item/parallel-navigation-item.twig' with {
+  active: true,
+  link_color: 'beaver-blue',
```
Comment (lal65) [inline packages/patternlab/source/patterns/molecules/parallel-navigation-item/parallel-navigation-item~related-active.twig:3]: Kill `link_color`.

File: packages/patternlab/source/patterns/molecules/parallel-navigation-item/parallel-navigation-item~related.twig [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{% include '@molecules/parallel-navigation-item/parallel-navigation-item.twig' with {
+  active: false,
+  link_color: 'beaver-blue',
```
Comment (lal65) [inline packages/patternlab/source/patterns/molecules/parallel-navigation-item/parallel-navigation-item~related.twig:3]: Kill `link_color`.

File: packages/patternlab/source/patterns/organisms/parallel-navigation/parallel-navigation~beaver-blue.twig [hunk 5]
```diff
@@ -1,22 +1,39 @@
...
-    { url: "#", title: "Health and Family Services (11)" },
-    { url: "#", title: "Homeland Security (16)" },
-    { url: "#", title: "Human Resources, Labor, and Employment Relations (8)" },
-    { url: "#", title: "Liberal Arts and Communications (16)" },
-    { url: "#", title: "Nursing (8)" },
-    { url: "#", title: "Technology, Sciences, and Math (25)" },
-    { url: "#", title: "Turfgrass (5)" }
-  ]
+  items: parallel_navigation_items
 } only %}
```
Comment (lal65) [inline packages/patternlab/source/patterns/organisms/parallel-navigation/parallel-navigation~beaver-blue.twig:39]: Kill example.

**Review by lal65 — CHANGES_REQUESTED** [review]
We can simplify things here.

**Comment by kjl16** [comment]
@lal65 , updated the parallel navigation item to use the shorthand setting of link classes.

**Review by lal65 — APPROVED** [review]

**Review by mdw15 — APPROVED** [review]

**Pull request merged**